wasm/main.py

```diff
@@ -3452,7 +3452,7 @@ def spec_binary_fN(raw, idx, N):
     for i in range(N // 8):
         bstar += bytearray([raw[idx]])
         idx += 1
-    return idx, spec_bytest_inv(ValType.get_float_type(N), bstar)  # bytearray(bstar)
+    return idx, spec_bytest_inv(ValType.get_float_type(BitSize(N)), bstar)  # bytearray(bstar)
```

Comment:
Yay for automatically forcing `N` to be one of the values, as part of the `enum` initialization :+1: